x11-plugins/wmacpi/files/wmacpi-2.3-makefile.patch

```diff
@@ -1,13 +1,13 @@
---- a/Makefile	2015-01-12 11:27:07.396319323 +0100
-+++ b/Makefile	2015-01-12 11:29:21.531298827 +0100
+--- a/Makefile
++++ b/Makefile
 @@ -13,8 +13,8 @@
  #OPT	= -pg -g
  
  CC	:= gcc
```

Comment:
this isn't an issue?

Reply:
No because the ebuild was calling `emake CC="$(tc-getCC)"`, but I've removed this now and switched to `tc-export CC` in src_configure